src/core/bitop.d

```diff
@@ -276,15 +291,21 @@ version (AnyX86)
  * They may be used to guarantee a write or read cycle occurs at a specified address.
  */
 
-ubyte  volatileLoad(ubyte * ptr);
-ushort volatileLoad(ushort* ptr);  /// ditto
-uint   volatileLoad(uint  * ptr);  /// ditto
-ulong  volatileLoad(ulong * ptr);  /// ditto
+pragma(DigitalMars_intrinsic, "ind")
```

Comment:
Hmm, so `dmd.yl2x` and `d.volatileLoad` or something...

With the current PR I have it's possible to do this:
```d
version (GDC)
{
    pragma(intrinsic, "gdc.volatileLoad", ...);
}
else version (DMD)
{
    pragma(intrinsic, "dmd.volatileLoad", ...);
}
```